src/detail-panels/account-header/handle-history/handle-history.jsx

```diff
@@ -60,79 +65,101 @@ function ChangeEntry({ date, handle, pds, lastHandle, lastPds, isCreated }) {
   const isMigratedToMushroom =
     isPdsChanged &&
     pds.endsWith('bsky.network') &&
-    lastPds && lastPds.endsWith('bsky.social');
-  const isMigratedExternally =
-    isPdsChanged &&
-    !pds.endsWith('bsky.network');
-  
-  if (isCreated) return (
-    <div className='event-entry'>
-      {date ? <FormatTimestamp className='event-timestamp' timestamp={date} /> : 'unknown date '}
-      <span className='event-label event-label-registered'>
-        {localise('registered ', { uk: 'зареєстровано ' })}
-      </span>
-      {
-        !handle ? undefined :
+    lastPds &&
+    lastPds.endsWith('bsky.social');
+  const isMigratedExternally = isPdsChanged && !pds.endsWith('bsky.network');
+
+  // getting elapsed time
+  function getRelativeTime(dateString) {
+    const newDate = new Date(dateString);
+    const now = new Date();
+    const timeDiffInMillis = newDate.getTime() - now.getTime();
+    const daysDiff = Math.round(timeDiffInMillis / (1000 * 60 * 60 * 24));
+
+    const inferred = new Intl.RelativeTimeFormat(undefined, {
+      numeric: 'auto',
+    });
+    return inferred.format(daysDiff, 'days');
+  }
```

Comment:
Rather than just having this logic here (and hard coded to days only) it would be nice to have the formatting updated in the common FormatTimestamp component

Reply:
> Rather than just having this logic here (and hard coded to days only) it would be nice to have the formatting updated in the common FormatTimestamp component

Okay cool I can move it into there! I was in there at first and then changed my mind because I was nervous about messing with the component lol..but I'll work on it today